core/chains/evm/types/types.go

```diff
@@ -44,7 +44,7 @@ type ORM interface {
 	CreateNode(data Node, qopts ...pg.QOpt) (Node, error)
 	DeleteNode(id int32, qopts ...pg.QOpt) error
 	GetNodesByChainIDs(chainIDs []utils.Big, qopts ...pg.QOpt) (nodes []Node, err error)
-	Node(id int32, qopts ...pg.QOpt) (Node, error)
+	NodeNamed(string, ...pg.QOpt) (Node, error)
```

Comment:
IIRC there is a uniqueness check for this in the config validation?

Reply:
Yeah, and we inherit the DB constraint on dump.